.travis.yml

```diff
@@ -94,7 +94,9 @@ install:
 
 before_script:
   - travis_retry ./testing/downloadSpark.sh $SPARK_VER $HADOOP_VER
-  - ./testing/setupLivy.sh
+  - if [[ -n $LIVY_VER ]]; then ./testing/downloadLivy.sh $LIVY_VER; fi
+  - if [[ -n $LIVY_VER ]]; then export LIVY_HOME=`pwd`/livy-server-$LIVY_VER; fi
+  - if [[ -n $LIVY_VER ]]; then export SPARK_HOME=`pwd`/spark-$SPARK_VER-bin-hadoop$HADOOP_VER; fi
```

Comment:
zeppelin-env.sh is used by zeppelin-server, while livy CI didn't start zeppelin-server. So I have to export it in .travis.yml explicitly and it won't affect the spark CI test as spark CI would always use zeppelin-env.sh